resources/lang/en/dashboard.php

```diff
@@ -83,19 +83,19 @@
         'component_statuses' => 'Component Statuses',
         'listed_group'       => 'Grouped under :name',
         'add'                => [
-            'title'   => 'Add a Component',
+            'title'   => 'Add a component',
             'message' => 'You should add a component.',
-            'success' => 'Component created.',
-            'failure' => 'Something went wrong with the component.',
+            'success' => 'Component has been added!',
+            'failure' => 'The component could not be added. Please try again.',
         ],
         'edit' => [
-            'title'   => 'Edit a Component',
-            'success' => 'Component updated.',
-            'failure' => 'Something went wrong with the component.',
+            'title'   => 'Edit a component',
+            'success' => 'Component has been updated!',
+            'failure' => 'The component could not be updated. Please try again.',
         ],
         'delete' => [
-            'success' => 'Component deleted.',
-            'failure' => 'The Component could not be deleted. Please try again.',
+            'success' => 'Component has been deleted!',
```

Comment:
Either "Component deleted" or "The component has been deleted", not "component has been deleted"